# Protein domain predictions  
Below are several websites that can be used to predict domains in proteins. 

Each site has different strengths and weaknesses depending on how the algorithms are written for the analysis.
Use multiple sources to check your protein and re-run the analysis occasionally to account for software updates.
Keep in mind some websites have their own engines and others combine multiple engines for their analysis.
See summary references at the bottom for more details.

1) NCBI  
Protein Blast: https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE=Proteins
Conserved domain database (CDD): https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi
    - Blast hits closely related proteins if interested in orthologs/homologs.
    - Blast links to CDD which provides information and references on domains.
    - CDD can also provide active sites, binding sites, and modification sites
    

1) Pfam  
http://pfam.xfam.org/
    - Predicts domains
    - Can search database of protein entries for precise maps that include transmembrane and low complexity regions
    
    
1) Smart  
http://smart.embl-heidelberg.de/
    - Predicts domains and other regions such as transmembrane regions
    - Loosely predicts interactions and orthology
    

1) InterPro  
https://www.ebi.ac.uk/interpro/
    - Matches protein families and domain superfamilies
    - Predicts domains, repeats, and signature matches
    - Annotates residues with interaction/active sites and key patterns


1) TMHMM Server (DTU Bioformatics)
v2: http://www.cbs.dtu.dk/services/TMHMM/
    - Predicts transmembrane helices


1) Signal P (DTU Bioinformatics)
http://www.cbs.dtu.dk/services/SignalP/
    - Predicts signal peptides
 
 
1) Phobius  
http://phobius.sbc.su.se/
    - Signal peptide and transmembrane domain predition


1) OrtholugeDB  
http://www.pathogenomics.sfu.ca/ortholugedb/  
    - Compare orthologs in multiple genomes  
    - Obtain orthologs between two genomes
    - Obtain orthologs for a single gene
   
   
1) Phyre 2  
http://www.sbg.bio.ic.ac.uk/phyre2/html/page.cgi?id=index  
    - Compares protein to known structures in database
    - Using comparisons, can predict interaction and binding sites
    



There are various websites that list resources or provide more background on the engines
- ExPASy Bioinformatics resource protal: https://www.expasy.org/resources/search/keywords:transmembrane  
- DTU Bioinformatics prediction servers: http://www.cbs.dtu.dk/services/  
- Online Analysis Tools: https://molbiol-tools.ca/Motifs.htm 
